README.md

```diff
@@ -28,4 +32,27 @@ planner.on("data", function (connection) {
 });
 ```
 
+## Multiple connection streams
+
+Instead of using one stream of connections, you can combine multiple streams as input by using `MergeStream`
+```javascript
+var connectionsStreams = [
+	[ 'stream1', connectionsReadStream1 ],
+	[ 'stream2', connectionsReadStream2 ],
+	...
+];
+
+var connectionsReadStream = new csa.MergeStream(connectionsStreams, query.departureTime);
+```
+
+You can add/remove/stop/start streams by setting an eventlistener on the MergeStream instance
+```javascript
+connectionsReadStream.on("data", function (connection) {
+	connectionsReadStream.addConnectionsStream(['newStream'], newConnectionsReadStream]);
+	connectionsReadStream.removeConnectionsStream('stream2');
+	connectionsReadStream.stopConnectionsStream('newStream');
+	connectionsReadStream.startConnectionsStream('newStream');
```

Comment:
what's the use of starting and stopping individual connections streams at this point? And if this would be useful, why not use pause and upause of the nodejs streams spec?